### ISSUE TITLE
GNOME libs and Sway surprisingly require Qt-basic

### ISSUE THREAD

**Issue by paulcarroty** [clearlinux/distribution#2979]
```
# swupd bundle-remove qt-basic

Bundle "qt-basic" is required by the following bundles:
 - desktop-gnomelibs
 - gnome-base-libs
 - sway
```

Which is wrong.

**Comment by fenrus75** [comment]
I'll buy the sway argument -- the gnome side not so much -- reality both qt
and gtk end up in pretty much any more complicated gui stacks

**Comment by paulcarroty** [comment]
Does Sway need [include(gnome-base-libs)](https://github.com/clearlinux/clr-bundles/blob/2cedfbcb01de115a3af6c37c44fed269156b137d/bundles/sway#L13C1-L14C1)?

**Comment by fenrus75** [comment]
if your real ask is "can we put sway on a diet"... lets make that the real
ask ;-)

I'd be happy to take a look at that

**Comment by paulcarroty** [comment]
Yeah, please do. My guess - clrbuilder has gone mad for a moment 'cause the last added Sway dependency - `xdg-desktop-portal-wlr` don't need qt.
